scripts/beachball/tagVNext.ts

```diff
@@ -0,0 +1,70 @@
+import { execSync } from 'child_process';
+import * as semver from 'semver';
+import * as tmp from 'tmp';
+import * as fs from 'fs';
+import { AllPackageInfo, getAllPackageInfo, isConvergedPackage } from '../monorepo/index';
+
+function tagPackages() {
+  const packagesToTag = getPackagesToTag();
+  const { path: npmrcPath, cleanup: cleanupTmpFiles } = createTmpNpmrc();
+
+  packagesToTag.forEach(pkg => {
+    tagPackage(pkg.name, pkg.version, npmrcPath);
+  });
+
+  cleanupTmpFiles();
+}
+
+function tagPackage(name: string, version: string, npmrcPath: string) {
+  const prerelease = semver.parse(version).prerelease;
+  if (prerelease.length == 0) {
+    return;
+  }
+
+  const prereleaseTag = prerelease[0];
+  const command = `npm dist-tag add ${name}@${version} ${prereleaseTag} --userconfig ${npmrcPath} --registry https://registry.npmjs.org/`;
+  console.log(command);
+  try {
+    const res = execSync(command);
```

Comment:
I wonder if it’s better to pipe the output so it gets logged even if the command fails. Right now I think it would get lost? Also want to make sure this is the right way to catch errors with this exec approach (I mix them up and can’t check right now since I’m not at a computer).

Reply:
that's a fair point I can just inherit stdio